scripts/vimpatch.lua

```diff
@@ -1,7 +1,7 @@
 --  Updates version.c list of applied Vim patches.
 --
 --  Usage:
---    VIM_SOURCE_DIR=~/neovim/.vim-src/ nvim -i NONE -u NONE --headless +'luafile ./scripts/vimpatch.lua' +q
+--    VIM_SOURCE_DIR=~/neovim/.vim-src/ nvim -V1 -es +'luafile ./scripts/vimpatch.lua' +q
```

Comment:
```suggestion
--    VIM_SOURCE_DIR=~/neovim/.vim-src/ nvim -V1 -i NONE -es +'luafile ./scripts/vimpatch.lua' +q
```

Reply:
gah, we should change `-es` to do that by default.

Reply:
We specifically document that it doesn't do that. :⁠-⁠)

Reply:
yes, but we should change it